src/main/groovy/org/fundacionjala/gradle/plugins/enforce/utils/salesforce/component/validators/files/DocumentSalesforceValidator.groovy

```diff
@@ -1,7 +1,13 @@
 package org.fundacionjala.gradle.plugins.enforce.utils.salesforce.component.validators.files
 
+import org.fundacionjala.gradle.plugins.enforce.utils.Constants
 import org.fundacionjala.gradle.plugins.enforce.utils.salesforce.MetadataComponents
 
+import java.nio.file.Paths
+
+/**
+ * This class defines how validate a document salesforce validator
```

Comment:
add "to" and remove "validator": This class defines how "to" validate a document salesforce